src/api/opencti/mod.rs

```diff
@@ -54,6 +60,8 @@ impl ApiOpenCTI {
     {
         use cynic::http::ReqwestExt;
         reqwest::Client::builder()
+            .timeout(Duration::from_secs(self.request_timeout))
```

Comment:
Have you checked how we handle the error produce by this method if a timeout occured?